addon/components/power-select-infinity/trigger.js

```diff
@@ -96,11 +100,30 @@ export default Component.extend({
         e.stopPropagation();
       }
 
+      if (e.keyCode === 8 && get(this, 'select.selected')) {
+          let select = get(this, 'select');
+          e.stopPropagation();
+          select.actions.select(null);
+          run.schedule('actions', null, select.actions.search);
+          run.schedule('actions', null, select.actions.open);
+      } else if (e.keyCode === 8 && get(this, 'select.searchText') <= 1) {
```

Comment:
is `select.searchText` a length here, or should this be `select.searchText.length`?